CHANGES.md

```diff
@@ -47,6 +47,33 @@
 * Fixed X (Java/Python) ([BEAM-X](https://issues.apache.org/jira/browse/BEAM-X)).
 -->
 
+# [2.22.0] - Unreleased
+
+## Highlights
+
+* New highly anticipated feature X added to Python SDK ([BEAM-X](https://issues.apache.org/jira/browse/BEAM-X)).
```

Comment:
Can you please remove these placeholders? ie keep the section headers but remove "feature X..."

Reply:
I don't see problems here. It's good to provide a sample when there is no entry. Contributors will overwrite the placeholder when they add something new.